### PR TITLE
Dockerfile: include spdx-licenses-list

### PR DESCRIPTION
**Pull request by Chocimier**
This is for linting license field on Travis.

One can also customize image as simple as
docker build "--build-arg=ADDINS=moreutils jq"

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,6 +17,7 @@ FROM scratch as stage1
 ARG REPOSITORY=https://alpha.de.repo.voidlinux.org
 ARG ARCH=x86_64
 ARG BASEPKG=base-minimal
+ARG ADDINS=spdx-licenses-list
```
Comment (the-maldridge) [inline Dockerfile]: You can't set this default, you'll break the normal non-masterdir containers.

Comment (Chocimier) [reply Dockerfile]: I do not understand what is the problem. Can you elaborate, or hint what i can do?

Comment (the-maldridge) [reply Dockerfile]: This dockerfile is used to build the masterdirs, as well as minimal containers.  By setting this default, the non-masterdir containers will also all include this package.  You need to either clear the default, or come up with some other way to make sure the containers don't get packages in them they shouldn't have.

**Comment by Chocimier** [comment]
Something like that maybe?

Didn't find how non-masterdirs are generated, but found something related to masterdirs. I didn't manage to build image with make either.

**Comment by the-maldridge** [comment]
Does that work?

**Comment by Chocimier** [comment]
When I run `docker build` manually, it works. However, as write previously, i do not know how are published images built, and couldn't figure out how to build them with use of Makefile.

**Comment by the-maldridge** [comment]
simply pass basepkg as base-minimal.  Strictly the makefile doesn't work to build the masterdirs either last I checked.

**Comment by Chocimier** [comment]
What is the scale of expected time to release images: week, month, half-year? I'm asking because it is more than month, I want to create some temporary way of linting licenses.

**Pull request merged**